tasks.py

```diff
@@ -31,7 +31,8 @@ def replace_char(string):
         '-n', '{}{}'.format(replace_char(scenario), '$' if 'Outline' not in scenario else ''),
         '-f', 'allure_behave.formatter:AllureFormatter',
         '-f', 'pretty',
-        '-o%allure_result_folder%', '--no-skipped']
```

Comment:
I was about to investigate this, from some reason the standard notation didn't work, however I am going to retest this now

Reply:
All is 👌